guides/common/modules/proc_upgrading-hosts-to-next-major-rhel-release.adoc

```diff
@@ -0,0 +1,43 @@
+[id="Upgrading_Hosts_to_Next_Major RHEL_release_{context}"]
+= Upgrading Hosts to Next Major {RHEL} Release
+
+You can use a job template to upgrade your {RHEL} hosts to the next major version.
```

Comment:
I'm thinking either also use "major release", or reword the title, anchor, file name, and include statement. I'm slightly leaning towards "major version".

Reply:
Hey @maximiliankolb just to confirm:
1. Do you mean changing the title to Upgrading Hosts to Next Major {RHEL} Version?
2. Do you want to drop {RHEL} from the title as well?
3. Replace `release` with `version` in the anchor and filename too.

Reply:
Gotcha! I have replaced "version" with "release".